sim/paladin/runes.go

```diff
@@ -84,6 +85,43 @@ func (paladin *Paladin) registerSheathOfLight() {
 	})
 }
 
+func (paladin *Paladin) registerShockAndAwe() {
+
+	if !paladin.hasRune(proto.PaladinRune_RuneCloakShockAndAwe) {
+		return
+	}
+
+	dep := paladin.NewDynamicStatDependency(
+		stats.Intellect, stats.SpellPower, 1.0,
+	)
+
+	shockAndAweAura := paladin.RegisterAura(core.Aura{
+		Label:    "Shock and Awe",
+		Duration: time.Second * 60,
+		ActionID: core.ActionID{SpellID: 462834},
+		OnGain: func(aura *core.Aura, sim *core.Simulation) {
+			paladin.EnableDynamicStatDep(sim, dep)
+		},
+		OnExpire: func(aura *core.Aura, sim *core.Simulation) {
+			paladin.DisableDynamicStatDep(sim, dep)
+		},
+	})
+	paladin.RegisterAura(core.Aura{
+		Label:    "Shock and Awe (rune)",
+		Duration: core.NeverExpires,
+		ActionID: core.ActionID{SpellID: 462834},
```

Comment:
🔨 Usually we try to not use action IDs on trigger auras like this since they're always active and kind of act as bloat in the UI